src/Viewport.ts

```diff
@@ -194,6 +194,8 @@ export class Viewport extends Container
     private _worldHeight?: number | null;
     private _disableOnContextMenu = (e: MouseEvent) => e.preventDefault();
 
+    public _wheelAxis?: 'all' | 'x' | 'y';
```

Comment:
If it's public, why the underscore? Should also make it non-optional and always have a value of one of the three, defaulting to 'all'. Possibly rename to something more generic like `scaleAxis` since scaling is not solely done by wheel, and wheel is not a global concept in this file but is handled by a plugin.